### ISSUE TITLE
Support php-cache

### ISSUE THREAD

**Issue by NiR-** [KnpLabs/Gaufrette#469]
See #309

**Comment by sprain** [comment]
Do you already have a plan what this should look like? I might be interested in helping out.

**Comment by NiR-** [comment]
Nope, no idea right now. It's meant to replace current `Apc` and `Cache` adapters. Feel free to make any proposition, we're eagerly looking out for new contributors :)
